spec/models/post_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
```

Comment:
Please ensure that there are unit tests for the `comment, and the `Like` models according to the project requirement.